src/encoded/static/components/forms/components/submission-fields.js

```diff
@@ -7,7 +7,7 @@ import url from 'url';
 import ReactTooltip from 'react-tooltip';
 import { DropdownButton, Button, MenuItem, Panel, Table, Collapse, Fade, Checkbox, InputGroup, FormGroup, FormControl } from 'react-bootstrap';
 import * as globals from './../../globals';
-import { ajax, console, object, isServerSide, animateScrollTo, Schemas } from './../../util';
+import { ajax, console, object, isServerSide, animateScrollTo, Schemas, fileUtil } from './../../util';
```

Comment:
You included fileUtil, but I don't see it used anywhere.

Reply:
Good call! Was rebasing this code off an old PR I made; not sure why that got in there in the first place